packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -141,7 +141,8 @@ function restLevelPackage(model: RLCModel, hasSamplesGenerated: boolean) {
         "@azure/core-paging": "^1.2.0"
       }),
       ...(hasLRO && {
-        "@azure/core-lro": "^2.2.0"
```

Comment:
Do we need to consider the flag `useLegancyLRO` in package.json?

Reply:
No need to, the package versions are backward compatible.